test/functional/test_framework/wallet.py

```diff
@@ -187,7 +186,11 @@ def sign_tx(self, tx, fixed_length=True):
         elif self._mode == MiniWalletMode.ADDRESS_OP_TRUE:
             tx.wit.vtxinwit = [CTxInWitness()] * len(tx.vin)
             for i in tx.wit.vtxinwit:
-                i.scriptWitness.stack = [CScript([OP_TRUE]), bytes([LEAF_VERSION_TAPSCRIPT]) + self._internal_key]
+                leaf_info = self._taproot_info.leaves["only-path"]
```

Comment:
I feel we should tie this `only-path` here to `taproot_construct` call in `address.py`, either by passing it as a parameter or creating it as a constant. Otherwise, it makes you wonder here at this line where did `only-path` come from.

Reply:
Yes, this is also good.